package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "elmi-to-json",
+  "version": "0.19.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "install": "binwrap-install",
+    "prepare": "binwrap-prepare",
+    "test": "binwrap-test",
+    "prepublish": "npm test"
+  },
+  "files": [
+    "index.js",
+    "bin/elmi-to-json"
+  ],
+  "bin": {
+    "elmi-to-json": "bin/elmi-to-json"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/stoeffel/elmi-to-json.git"
+  },
+  "author": "",
```

Comment:
description and author probably need to be filled in before npm will allow it to be published.